internal/app/secretless/handlers/http/conjur.go

```diff
@@ -38,7 +38,14 @@ func (h ConjurHandler) Authenticate(values map[string][]byte, r *http.Request) e
 
 // ConjurHandlerFactory instantiates a handler given HandlerOptions
 func ConjurHandlerFactory(options plugin_v1.HandlerOptions) plugin_v1.Handler {
-	return &ConjurHandler{
+	handler := &ConjurHandler{
 		BaseHandler: plugin_v1.NewBaseHandler(options),
 	}
+
+	// Force instantiate the Conjur provider so we can use the access token
+	if handler.Resolver != nil {
```

Comment:
Can you add a comment here for clarity that nothing will happen unless Secretless is configured to authenticate to Conjur?